pyproject.toml

```diff
@@ -1,6 +1,77 @@
+[project]
+name = "Flask-DebugToolbar"
+version = "0.15.1"
+description = "A toolbar overlay for debugging Flask applications."
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+author = [{ name = "Michael van Tellingen" }]
+maintainers = [{ name = "Pallets Ecosystem", email = "contact@palletsprojects.com" }]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Probably about time we move this out of Beta status 😁

Reply:
Yes, we should also think about what 1.0 looks like.